Documentation/ceph-nfs-crd.md

```diff
@@ -442,3 +456,98 @@ configuration objects.
+
+
+## Ceph CSI NFS provisioner and NFS CSI driver
+> **EXPERIMENTAL**: this feature is experimental, and we do not guarantee upgrade support to future versions
+
+In version 1.9.1, Rook is able to deploy the experimental NFS Ceph CSI driver. This requires Ceph
+CSI version 3.6.0 or above. We recommend Ceph v16.2.7 or above. Ceph CSI's
+[support matrix](https://github.com/ceph/ceph-csi#support-matrix) has more detailed information.
+
+For this section, we will refer to Rook's deployment examples in the
+[deploy/examples](https://github.com/rook/rook/tree/{{ branchName }}/deploy/examples) directory.
+
+Rook will only deploy the Ceph CSI NFS provisioner and driver components when the
+`ROOK_CSI_ENABLE_NFS` config is set to `"true"` in the `rook-ceph-operator-config` configmap. Change
+the value in your manifest, or patch the resource as below.
+```sh
+kubectl --namespace rook-ceph patch configmap rook-ceph-operator-config --type merge --patch '{"data":{"ROOK_CSI_ENABLE_NFS": "true"}}'
+```
+
+In order to create NFS exports via the CSI driver, you must first create a CephFilesystem to serve
+as the underlying storage for the exports, and you must create a CephNFS to run an NFS server that
+will expose the exports.
+
+From the examples, `filesystem.yaml` creates a CephFilesystem called `myfs`, and `nfs.yaml` creates
+an NFS server called `my-nfs`.
+
+You do not need to enable Ceph's Rook orchestrator mgr module if you are using Ceph v16.2.8 or
+higher. For Ceph v17.2.0, the orchestrator module must remain disabled. If it is enabled, you can
+disable it using the snippet below from the toolbox.
+```console
+ceph orch set backend ""
+ceph mgr module disable rook
+```
+
+You must also create a storage class. Ceph CSI is designed to support any arbitrary Ceph cluster,
+but we are focused here only on Ceph clusters deployed by Rook. Let's take a look at a portion of
+the example storage class found at `deploy/examples/csi/nfs/storageclass.yaml` and break down how
+the values are determined.
+
+```yaml
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: rook-nfs
+provisioner: rook-ceph.nfs.csi.ceph.com # [1]
+parameters:
+  nfsCluster: my-nfs # [2]
+  server: rook-ceph-nfs-my-nfs-a # [3]
+  clusterID: rook-ceph # [4]
+  fsName: myfs # [5]
+  pool: myfs-replicated # [6]
+
+  # [7] (entire csi.storage.k8s.io/* section immediately below)
+  csi.storage.k8s.io/provisioner-secret-name: rook-csi-cephfs-provisioner
+  csi.storage.k8s.io/provisioner-secret-namespace: rook-ceph
+  csi.storage.k8s.io/controller-expand-secret-name: rook-csi-cephfs-provisioner
+  csi.storage.k8s.io/controller-expand-secret-namespace: rook-ceph
+  csi.storage.k8s.io/node-stage-secret-name: rook-csi-cephfs-node
+  csi.storage.k8s.io/node-stage-secret-namespace: rook-ceph
+
+# ... some fields omitted ...
+```
+
+1. `provisioner`: **rook-ceph**.nfs.csi.ceph.com because **rook-ceph** is the namespace where the
```

Comment:
nit: The example values are already seen above, seems like no need to repeat them in this section describing the settings. But it's just a style suggestion, take it or leave it... 
```suggestion
1. `provisioner`: the namespace where the CephCluster is installed
```

Reply:
I think it's clearer to leave it since "the namespace where the CephCluster is installed" doesn't explain that only a small part of the value is taken from that parameter.